hieradata_aws/common.yaml

```diff
@@ -755,6 +755,43 @@ govuk_jenkins::jobs::integration_deploy::applications: *deployable_applications
 govuk_jenkins::jobs::deploy_lambda_app::lambda_apps:
   - 'email_alert_notifications'
 
+govuk_jenkins::config::banner_string: 'AWS Integration'
+govuk_jenkins::config::banner_colour_background: '#005EA5'
+govuk_jenkins::config::banner_colour_text: 'white'
+govuk_jenkins::config::theme_colour: '#005EA5'
+govuk_jenkins::config::theme_text_colour: 'white'
+govuk_jenkins::config::theme_environment_name: 'AWS Integration'
+
```

Comment:
These names, where do they get changed per environment now?

Reply:
The default config shows Integration. The overrides happen in `staging.yaml` and `production.yaml` respectively.

Reply:
Sorry, it helps if I read. :-) `"If we move the default configuration to common.yaml, then we can overwrite it using staging.yaml, which is clearer to understand."`